src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/JourneyMapLocationController.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.controller
+
+import org.springframework.stereotype.Controller
+import org.springframework.ui.ModelMap
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.ModelAttribute
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.SessionAttributes
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Supplier
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.JourneyMapLocationService
+
+@Controller
+@SessionAttributes("supplier")
+class JourneyMapLocationController(private val service: JourneyMapLocationService) {
+
+  @GetMapping("/map-pickup-location/{journey-id}")
```

Comment:
So I thought we would drive the mapping here by the journey id.  Does this makes sense in this context.